06_puppy/kasunfdo/main_test.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := "Puppy(1) added to store\n"
+	actual := buf.String()
+
+	assert.Equalf(t, expected, actual, "Unexpected output")
```

Comment:
No need for this failure message, testify has you covered unless you print something specific with context that makes sense to you.